### DISCUSSION TITLE
Wifi works but regular LAN connection doesn't

### DISCUSSION THREAD

**Discussion by dtsmarin** [apohl79/audiogridder#642] [Help]
I'm using a regular LAN connection with static IPs between a Mac and two PCs for Vienna Ensemble Pro via a switch. I don't have another port available so I can only use this connection. I have to use both software for different tasks.

Does AG support this type of connection or do I have to use a crossover cable which may or may not work for VEP?

I can ping to the other computer but AG doesn't connect when I type in the IP address. 

My apologies if this has been covered extensively before.

**Comment by CaBuffalo** [comment]
You only need to use crossover cable if you connect two PCs directly. No,
you don't need it if there is a switch / router between them.

On Wed, Jul 14, 2021 at 3:29 PM dtsmarin ***@***.***> wrote:

> I'm using a regular LAN connection with static IPs between a Mac and two
> PCs for Vienna Ensemble Pro via a switch. I don't have another port
> available so I can only use this connection. I have to use both software
> for different tasks.
>
> Does AG support this type of connection or do I have to use a crossover
> cable which may or may not work for VEP?
>
> I can ping to the other computer but AG doesn't connect when I type in the
> IP address.
>
> My apologies if this has been covered extensively before.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/642>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6A65RON5D6MIXJJ3XLTXYFUJANCNFSM5AMK5U6Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by dtsmarin** [reply]
Sorry it turns out it my LAN network was classified as a Public network and I had forgot to check the Firewall rules. Now it works great! But I need to investigate how to change the LAN network to private anyways.
